tasks/47_string-expansion/string-expansion.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * stringExpansion - the numeric values represent the
+ * occurrence of each letter preceding that numeric value
+ * @example
+ * stringExpansion('3D2a5d2f');
+ * // return 'DDDaadddddff'
+ * @param {String} str
+ * @return {String}
+ */
+export function stringExpansion(str) {
+  let result = '';
+  let repeatCount = 0;
+  for (let i = 0; i < str.length; i++) {
+    if (isNotNumber(str[i])) {
```

Comment:
Looks like `isNotNumber()` function just parse string to number and checks if it is `NaN`. But later in the code, you parse `str[i]` again. Why not parse it in the beginning of the loop and save in variable, use `isNaN` here and later access it via variable, not parsing again?